docs/manage/data/query.md

```diff
@@ -0,0 +1,150 @@
+
+In addition, if you intend to directly query tabular data from an MQL-compatible client, you must also [configure direct data query](/services/data/configure-data-query/).
+
+## Query tabular data in the Viam app
+
+Once you have synced tabular data to the Viam app, you can run SQL or MQL queries against your synced data from the [**Query** subtab](https://app.viam.com/data/query) under the **Data** tab in the Viam app.
+You must have the [owner](/manage/fleet/#permissions) role in order to query data in the Viam app.
+
+1. Navigate to the [**Query** subtab](https://app.viam.com/data/query).
+
+1. Select either `SQL` or `MQL` from the **Query mode** dropdown menu on the right-hand side, then enter your query using the respective syntax for the language you have selected in the text area at the top of your screen.
+   For example:
+
+   - The following shows a SQL query that filters by the component name `my-ultrasonic-sensor` and limits the returned results to 5:
+
+     {{< imgproc src="/manage/data/query-ui-sql.png" alt="Viam App Data Query tab with a SQL query shown" resize="800x" >}}
+
+   - The following shows the same search using MQL syntax:
+
+     {{< imgproc src="/manage/data/query-ui-mql.png" alt="Viam App Data Query tab with an MQL query shown" resize="800x" >}}
+
+1. Click **Run query** when ready to perform your query and get matching results.
+   Query results are displayed as a [JSON array](https://json-schema.org/understanding-json-schema/reference/array) below your query.
+   For example:
+
+   - The following shows a SQL query that filters by component name and specific column names, and its returned results:
+
+     {{< imgproc src="/manage/data/query-ui-results.png" alt="Viam App Data Query tab with a SQL query shown and results shown below including two matching records" resize="800x" >}}
+
+   - The following shows a SQL query that returns a count of records matching the search criteria:
+
+     {{< imgproc src="/manage/data/query-ui-numreadings.png" alt="Viam App Data Query tab with a SQL query shown with the resulting count of matching records displayed below" resize="800x" >}}
+
+For more information on MQL syntax, see the [MQL (MongoDB Query Language)](https://www.mongodb.com/docs/manual/tutorial/query-documents/) documentation.
+
+## Query tabular data using the API
+
+Once you have synced tabular data to the Viam app, you can use the following data management API methods in your `go` code to query tabular data using either SQL or MQL:
+
+- Use the [`TabularDataBySQL`](/services/data/#tabulardatabysql) method to query synced tabular data using SQL.
+- Use the [`TabularDataByMQL`](/services/data/#tabulardatabymql) method to query synced tabular data using the [MQL (MongoDB Query Language)](https://www.mongodb.com/docs/manual/tutorial/query-documents/).
+
+These APIs are currently implemented only in the [Go SDK](https://pkg.go.dev/go.viam.com/rdk).
```

Comment:
They're actually only currently implemented in the TypeScript SDK: https://ts.viam.dev/classes/dataApi.TabularDataByMQLRequest.html and https://ts.viam.dev/classes/dataApi.TabularDataBySQLRequest.html.

It will be included in the Flutter SDK in its next stable release.